src/components/NcAppSidebar/NcAppSidebar.vue

```diff
@@ -732,7 +736,7 @@ export default {
 			 *
 			 * @type {Event}
 			 */
-			this.$emit('figure-click', e)
+			this.$attrs.onFigureClick(e)
```

Comment:
We still should emit this event, otherwise it won't be visible in `vue-devtools`